### ISSUE TITLE
Enable Rapid's 'MapRoulette tasks' option url option - &maproulette=<challenge id>

### ISSUE THREAD

**Issue by atiannicelli** [maproulette/maproulette3#2513]
OK, here is what I'm looking for... Rapid has new functionality that allows a user to enter the challenge ID for a MapRoulette challenge to see other tasks in the area. What I would like MapRoulette to do when I click on "Edit in Rapid" for a specific task is to add the options to enable that "mapRoulette Tasks" option and add the challenge ID that we are coming from. So what this would do is it would make it possible for a user that is coming from MapRoulette to Rapid to see all the other MpaRoulette tasks for that challenge in the area. The user could fix them while they are here before going back to MapRoulette to get the next random task.

To do this it would be very easy. When clicking on the "edit in Rapid" button just add the following test to the rapid URL
&maproulette=<challenge id>
where the <challenge id> is the challenge that I am currently working on. I believe that you already add the &id=<featureID> option to the URL so it should be an easy slam dunk. All this would do is enable Rapid to show other tasks in this challenge. The user doesn't have to actually fix any other if there are any, but it would be just nice to see if others are in the area.

**Comment by atiannicelli** [comment]
Another more generic way to do this would be to add a new "Editor Configuration" option to the challenge's edit page. The option would allow you to add editor specific url options. So for instance for challenge number 12345 the user could add an option in the challenge configuration to add `&maproulette=12345` to the url whenever you open the rapid link. This could be very generic and would allow for users to add other weird options to their favorite editor.

**Comment by mvexel** [comment]
I like both options here. The editor configuration requires a little thought so as to prevent less technically inclined users to accidentally mess with it and get unpredictable outcomes. 

If we add `maproulette=<challenge_id>` to the Rapid URL, will that enable the MapRoulette tasks overlay in Rapid if it is not already @bhousel? If not, is that an option we could add to the Rapid URL params?

**Comment by mvexel** [comment]
Rapid does already support `maproulette=<challenge_id>`  - @maproulette/mr3 any considerations for passing this URL param on our side?

**Comment by jake-low** [comment]
Nope, seems like a great idea and Rapid already has the URL parameter we need to enable this. I'll look into adding this functionality soon.
